transcendence-app/src/chat/infrastructure/db/chatmember.repository.ts

```diff
@@ -1,7 +1,12 @@
-import { ChatMember, ChatMemberWithUser } from '../../chatmember.domain';
+import { ChatMemberWithUser } from '../../chatmember.domain';
+import { ChatMemberEntity } from './chatmember.entity';
 
 export abstract class IChatMemberRepository {
-  abstract add(chatMember: ChatMember): Promise<ChatMember | null>;
+  abstract add(chatMember: ChatMemberEntity): Promise<ChatMemberEntity | null>;
```

Comment:
In user, the add function takes a User (from domain) and returns a Promise<User|null>. Why here you change this to use entity instead?

Reply:
We might want to make some changes to our repository methods.
In the current implementation, we return an object with the same properties as the entity/domain. They seem interchangeable.
We could return an entity instance, which we could serialize using `ClassSerializerInterceptor`.

Reply:
I've created a new task for this #301

Reply:
I made a POC with repositories that return entities instances and no domains #303